File: service/src/main/java/by/rekuts/travelagency/service/HotelService.java
package by.rekuts.travelagency.service;

import by.rekuts.travelagency.domain.Hotel;

/**
 * Interface HotelService concerns to Service layer.
 * Realisations of this interface has access to DAO layer
 * in the face of HotelRepository interface and could call its methods.
 */
public interface HotelService extends CrudService<Hotel> {
}

File: service/src/main/scala/by/rekuts/travelagency/service/HotelService.scala
package by.rekuts.travelagency.service

import by.rekuts.travelagency.domain.Hotel

/**
  * Trait HotelService concerns to Service layer.
  * Realisations of this trait has access to DAO layer
  * in the face of HotelRepository trait and could call its methods.
  */
trait HotelService extends CrudService[Hotel]
